frontend/src/metabase/actions/containers/ActionCreator/tests/ActionCreator-QueryActions.unit.spec.ts

```diff
@@ -1,14 +1,16 @@
-import { screen, getIcon, queryIcon } from "__support__/ui";
+import userEvent from "@testing-library/user-event";
+import { screen, getIcon, queryIcon, within } from "__support__/ui";
 
 import {
   createMockActionParameter,
+  createMockCard,
   createMockQueryAction,
 } from "metabase-types/api/mocks";
 
 import { setup } from "./common";
 
 describe("ActionCreator > Query Actions", () => {
-  describe("new action", () => {
+  describe("New Action", () => {
```

Comment:
Should you change https://github.com/metabase/metabase/blob/7b47197d561ed51a8765b4c7d75e8fea23001a4c/frontend/src/metabase/actions/containers/ActionCreator/tests/ActionCreator-QueryActions.unit.spec.ts#L113

to `Editing Action` too? Since it's at the same level as this `describe` block, I think their names should be consistent.

Reply:
changed